components/List/index.js

```diff
@@ -0,0 +1,20 @@
+import React, {Component} from 'react'
+import {StyleSheet, View} from 'react-native'
+
+export default class List extends Component {
+    render() {
```

Comment:
if you ever find yourself writing a component with only a `render()` method, it should probably be a pure functional one